CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.2.0]
+### Added
+*   Payment type ApplePay.
```

Comment:
Please make a full sentence here, so we can use them for the release mails.
e.g. "Introduced the payment type ApplePay."